jgrapht-core/src/main/java/org/jgrapht/experimental/alg/color/BrownBacktrackColoring.java

```diff
@@ -28,12 +28,15 @@
  * @param <V> the graph vertex type
  * @param <E> the graph edge type
  * 
- * @author micha
+ * @author Michael Behrisch
  */
 public class BrownBacktrackColoring<V, E>
```

Comment:
I have updated the TODO page in the wiki with most of these changes! 

I completely agree about the experimental package. They should either be improved and moved out of the experimental, or deleted completely. The DAG class is the most important which should be absolutely retained after some revisions.